lib/topic_view.rb

```diff
@@ -602,6 +602,14 @@ def setup_filtered_posts
     @contains_gaps = false
     @filtered_posts = unfiltered_posts
 
+    if SiteSetting.ignore_user_enabled
+      if IgnoredUser.where(user_id: @user.id).present?
```

Comment:
Do we need to do this request?

Reply:
Not strongly opinionated on it, but I guess it is useful since if there are no ignored users at all for the current signed in User, we avoid appending the below slightly more complicated `where` query to the `@filtered_posts`

Reply:
The fastest queries are the ones you don't make ;)

Reply:
Makes sense 👍